docs/docs/gestures/gesture.md

```diff
@@ -0,0 +1,93 @@
+---
+id: gesture
+title: Gesture
+sidebar_label: Gesture
+sidebar_position: 2
+---
+
+`Gesture` is the object that allows you to create and compose gestures.
+
+## Reference
+
+```jsx
+import { GestureDetector, Gesture } from 'react-native-gesture-handler';
+
+function App() {
+  // highlight-next-line
+  const tap = Gesture.Tap();
+
+  return (
+    <GestureDetector gesture={tap}>
+      <Animated.View />
+    </GestureDetector>
+  );
+}
+```
+
+### Gesture.Tap()
+
+Creates a new instance of [`TapGesture`](/docs/gestures/tap-gesture.md) with its default config and no callbacks.
+
+### Gesture.Pan()
+
+Creates a new instance of [`PanGesture`](/docs/gestures/pan-gesture.md) with its default config and no callbacks.
+
+### Gesture.LongPress()
+
+Creates a new instance of [`LongPressGesture`](/docs/gestures/long-press-gesture.md) with its default config and no callbacks.
+
+### Gesture.Fling()
+
+Creates a new instance of [`FlingGesture`](/docs/gestures/fling-gesture.md) with its default config and no callbacks.
+
+### Gesture.Pinch()
+
+Creates a new instance of [`PinchGesture`](/docs/gestures/pinch-gesture.md) with its default config and no callbacks.
+
+### Gesture.Rotation()
+
+Creates a new instance of [`RotationGesture`](/docs/gestures/rotation-gesture.md) with its default config and no callbacks.
+
+### Gesture.ForceTouch()
+
+Creates a new instance of [`ForceTouchGesture`](/docs/gestures/force-touch-gesture.md) with its default config and no callbacks.
+
+### Gesture.Manual()
+
+Creates a new instance of [`ManualGesture`](/docs/gestures/manual-gesture.md) with its default config and no callbacks.
+
+### Gesture.Native()
+
+Creates a new instance of [`NativeGesture`](/docs/gestures/native-gesture.md) with its default config and no callbacks.
```

Comment:
It looks like `Hover` is missing there, could you add it and scold the right person for it not being there in the first place?